src/jaegertracing/testutils/EnvVariable.cpp

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2017 Uber Technologies, Inc.

```

Comment:
please change to `2019 The Jaeger Authors`

Reply:
Updated.